systems/WikibaseClient/06-Runtime_View.md

```diff
@@ -1,8 +1,26 @@
 # Runtime View
 
-## Runtime Scenario 1
+## UpdateRepoHookHandler
 
-## Runtime Scenario 2
+```mermaid
+sequenceDiagram
+    autonumber
+    alt is move
+        MediaWiki Page->>+UpdateRepo-HookHandler: PageMoveComplete (hook)
+    else is delete
+        MediaWiki Page->>+UpdateRepo-HookHandler: ArticleDeleteComplete (hook)
+    end
+    UpdateRepo-HookHandler->>UpdateRepo-HookHandler: Check if altered pages are connected to Items
+    opt Not connected to Items
+        UpdateRepo-HookHandler-->>MediaWiki Page: Return control flow
+    end
+    UpdateRepo-HookHandler->>+Repo MediaWiki JobQueue: Notify Repo of the changes (job)
+    opt Repo Notified
+        UpdateRepo-HookHandler-->>MediaWiki Page: Return control flow
+    end
+```
+
+## Data Bridge
```

Comment:
This will create merge conflicts with the data bridge sequence diagram PR

Reply:
Fixed!